src/pudl/io_managers.py

```diff
@@ -327,9 +327,13 @@ def handle_output(
             )
         elif isinstance(obj, pl.LazyFrame):
             logger.warning(
-                "PudlParquetIOManager currently does not do any schema enformcement for polars LazyFrames"
+                "PudlParquetIOManager currently does not do any schema enforcement for polars LazyFrames"
+            )
+            obj.sink_parquet(
+                parquet_path,
+                engine="streaming",
+                row_group_size=100_000,
             )
```

Comment:
I tested doing some basic schema enforcement here, which seems to work fine, although if it's not doing everything in `Resource.enforce_schema`. Note: this has to be paired with another change to the `FIELD_DTYPES_POLARS` dict to fix an issue with `Datetime` type, that we should probably fix anyway.

```suggestion
            )

            schema = Resource.from_id(table_name).to_polars_dtypes()
            (  
                obj.match_to_schema(schema).sink_parquet(
                    parquet_path,
                    engine="streaming",
                    row_group_size=100_000,
                )
            )
```